service/service.go

```diff
@@ -99,11 +99,14 @@ func (svc *Service) Init(ctx context.Context, cfg *config.Config, serviceList *E
 
 	if cfg.IsPublishing {
 		svc.Cache.CensusTopic.AddUpdateFunc(cache.CensusTopicID, cachePrivate.UpdateCensusTopic(ctx, cfg.ServiceAuthToken, clients.Topic))
-		svc.Cache.DataTopic.AddUpdateFuncs(cachePrivate.UpdateDataTopics(ctx, cfg.ServiceAuthToken, clients.Topic))
+		if svc.Config.EnableTopicAggregationPages {
```

Comment:
I would suggest a component level test to ensure no requests go to the Topic API if it's switched off. Or I guess maybe only to the root topic. Something to think about. Not dead set on it.